MoreLinq.Test/StartsWithTest.cs

```diff
@@ -61,19 +61,20 @@ public bool StartsWithWithStrings(string first, string second)
         [Test]
         public void StartsWithReturnsTrueIfBothEmpty()
         {
-            new int[0].StartsWith(new int[0]);
+            Assert.True(new int[0].StartsWith(new int[0]));
         }
 
         [Test]
         public void StartsWithReturnsFalseIfOnlyFirstIsEmpty()
         {
-            new int[0].StartsWith(new[] {1,2,3});
+            Assert.False(new int[0].StartsWith(new[] {1,2,3}));
         }
 
-        [Test]
-        public void StartsWithReturnsFalseIfOnlySecondIsEmpty()
+        [TestCase("", "", Result = true)]
+        [TestCase("1", "", Result = true)]
+        public bool StartsWithReturnsTrueIfSecondIsEmpty(string first, string second)
         {
-            new[] {1,2,3}.StartsWith(new int[0]);
+            return first.StartsWith(second);
```

Comment:
Hmmm, I bet this does not call your implementation of `StartsWith` but [`String.StartsWith`](https://msdn.microsoft.com/en-us/library/baketfxw(v=vs.110).aspx), so while the test may be passing, it's not testing the right thing!